modules/user-agent-overview.adoc

```diff
@@ -23,11 +23,11 @@ For example, when:
 * `<command>` = `oc`
```

Comment:
@adambkaplan This content is similarly not included in the published docs at this time. We should update it in case someone decides to re-add it, eventually.